deployment/common/changeset/transfer_to_mcms_with_timelock_test.go

```diff
@@ -78,3 +80,42 @@ func TestTransferToMCMSWithTimelock(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, e.Chains[chain1].DeployerKey.From, o)
 }
+
+func TestRevokeTimelockDeployer(t *testing.T) {
+	lggr := logger.TestLogger(t)
+	e := memory.NewMemoryEnvironment(t, lggr, 0, memory.MemoryEnvironmentConfig{
+		Chains: 1,
+		Nodes:  1,
+	})
+	chain1 := e.AllChainSelectors()[0]
+	e, err := ApplyChangesets(t, e, nil, []ChangesetApplication{
+		{
+			Changeset: WrapChangeSet(DeployMCMSWithTimelock),
+			Config: map[uint64]types.MCMSWithTimelockConfig{
+				chain1: proposalutils.SingleGroupTimelockConfig(t),
+			},
+		},
+	})
+	require.NoError(t, err)
+	addrs, err := e.ExistingAddresses.AddressesForChain(chain1)
+	require.NoError(t, err)
+
+	state, err := MaybeLoadMCMSWithTimelockChainState(e.Chains[chain1], addrs)
+	require.NoError(t, err)
+
+	tl := state.Timelock
+	require.NotNil(t, tl)
+
+	// WHAT AM I DOING HERE? HOW DO I TEST?
+	adminRole, err := tl.ADMINROLE(nil)
+	require.NoError(t, err)
+
+	r, err := tl.GetRoleAdmin(&bind.CallOpts{}, adminRole)
+	require.NoError(t, err)
+	h := hex.EncodeToString(r[:])
+	t.Logf("admin role: %s", h)
+	a := common.BytesToAddress(r[:])
+
+	require.Equal(t, a, e.Chains[chain1].DeployerKey.From)
```

Comment:
this fails. 

i thought that the admin would be the deployer key and we have to change it

what's going on?

Reply:
GetRoleAdmin will just return the roleID. Its annoying but this is how you get the members that have the role https://github.com/smartcontractkit/chainlink/blob/88a0338576dad34068f8ee278a5cf5ac0b8870ad/deployment/common/view/v1_0/mcms.go#L87